spec/fixtures/in_fine_month_1000.0_12.0_12_0_20200101_with_initial_values.csv

```diff
@@ -0,0 +1,12 @@
+1,2020-01-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,20.0,20.0,1000.0,1000.0
+2,2020-02-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,40.0,20.0,1000.0,1000.0
+3,2020-03-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,60.0,20.0,1000.0,1000.0
+4,2020-04-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,80.0,20.0,1000.0,1000.0
+5,2020-05-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,100.0,20.0,1000.0,1000.0
+6,2020-06-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,120.0,20.0,1000.0,1000.0
+7,2020-07-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,140.0,20.0,1000.0,1000.0
+8,2020-08-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,160.0,20.0,1000.0,1000.0
+9,2020-09-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,180.0,20.0,1000.0,1000.0
+10,2020-10-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,200.0,20.0,1000.0,1000.0
+11,2020-11-01,1000.0,1000.0,20.0,0.0,0.0,0.0,20.0,0.0,0.0,220.0,20.0,1000.0,1000.0
+12,2020-12-01,1000.0,0.0,1020.0,0.0,0.0,0.0,1020.0,1000.0,1000.0,1240.0,2020.0,1000.0,0
```

Comment:
No newline at end of file

(configure ton éditeur pour l'ajouter automatiquement, ça évite de casser des trucs)